qiita_db/metadata_template/prep_template.py

```diff
@@ -874,8 +882,9 @@ def _get_predecessors(workflow, node):
                         for iname, dname in rp.items():
                             if dname not in init_artifacts:
                                 msg = (f'Missing Artifact type: "{dname}" in '
-                                       'this preparation; are you missing a '
-                                       'step to start?')
+                                       'this preparation; this might be due '
+                                       'to missing steps or not having the '
+                                       'correct raw data.')
```

Comment:
Raises option c seems cryptic after briefly reviewing the function. It's not part of the PR though, so I'll raise it as separate and low-priority issue.

Reply:
This function adds a default workflow to a preparation so IMOO the error message makes sense.

Reply:
ok.